facebook-commerce.php

```diff
@@ -1398,6 +1416,41 @@ function update_product_group( $woo_product ) {
 		*/
 	}
 
+
+	/**
+	 * Determine if there matching variation to the default attributes
+	 *
+	 * @param WC_Facebook_Product $woo_product
+	 *
+	 * @return WC_Product_Variation|null
+	 */
+	protected function get_product_group_default_variation( $woo_product ) {
+
+		$default_attributes = $woo_product->woo_product->get_default_attributes( 'edit' );
+
+		if ( empty( $default_attributes ) ) {
+			return null;
+		}
+
+		$default_variation  = null;
+		$product_variations = $woo_product->woo_product->get_available_variations( 'objects' );
```

Comment:
if I am not mistaken in old (yet supported) versions of WooCommerce (see for example WC 3.0.x) this method would only return an array and has no support for `objects` - in this case the foreach loop below would produce a PHP error; we should ensure this not to be a possibility and watch out for types

Reply:
this 648ea1f should take care of that